docs/content/python/new-roles/examples.md

```diff
@@ -27,12 +28,13 @@ To define a role for another resource type, we add another `resource`
 rule to our policy.
 
 {{< literalInclude
-    path="examples/gitclub-sqlalchemy-flask-react/backend/app/authorization.polar"
-    from="docs: begin-repo-resource"
-    to="docs: end-repo-resource"
-    gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
-    linenos=true
->}}
+path="examples/gitclub-sqlalchemy-flask-react/backend/app/authorization.polar"
+from="docs: begin-repo-resource"
+to="docs: end-repo-resource"
+gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
+linenos=true
+
+> }}
```

Comment:
is this valid after going through the autocorrect thresher?

Reply:
ugh let me just rebase